<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserPlugin::Internal::UserViewer_RightsUI</class>
 <widget class="QWidget" name="UserPlugin::Internal::UserViewer_RightsUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_25">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>User manager rights</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_28">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Drugs dosage rights</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_27">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Agenda rights</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="UserPlugin::Internal::UserRightsWidget" name="userManagerRightsListWidget"/>
   </item>
   <item row="1" column="1">
    <widget class="UserPlugin::Internal::UserRightsWidget" name="drugsRightsListWidget"/>
   </item>
   <item row="1" column="2">
    <widget class="UserPlugin::Internal::UserRightsWidget" name="agendaRightsWidget"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_19">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Medical rights</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_26">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Paramedical rights</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2" rowspan="2">
    <widget class="QLabel" name="label_29">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Administrative rights</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="UserPlugin::Internal::UserRightsWidget" name="administrativeRightsWidget"/>
   </item>
   <item row="4" column="1">
    <widget class="UserPlugin::Internal::UserRightsWidget" name="paramedicalRightsWidget"/>
   </item>
   <item row="4" column="0">
    <widget class="UserPlugin::Internal::UserRightsWidget" name="medicalRightsListWidget"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UserPlugin::Internal::UserRightsWidget</class>
   <extends>QListWidget</extends>
   <header location="global">usermanagerplugin/widgets/userrightswidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
